release-notes/Cloud/DCP_change_log.md

```diff
@@ -6,6 +6,25 @@ uid: DCP_change_log
 
 The dataminer.services platform gets updated continuously. This change log can help you trace when specific features and changes have become available.
 
+#### 2 March 2023 - Integrate your DataMiner System with Microsoft Teams using DataMiner Automation [ID_35799]
+
+From now on you can easily integrate your DataMiner System with Microsoft Teams using DataMiner Automation.
+
+The following features become availabe as from today:
+- Creating Teams
+- Creating Channels
+- Add members or owners to your Teams
+- Sending notifications in the created Channels in the name of the DataMiner Teams bot 
+- Creating a private chat between someone and the DataMiner Teams bot
+- Fetching a private chat between someone and the DataMiner Teams bot
+- Sending notifications in those private chats in the name of the DataMiner Teams bot 
+
+In an Automation script you can use [the DcpChatIntegrationHelper NuGet](https://www.nuget.org/packages/Skyline.DataMiner.DcpChatIntegrationHelper) to easily interact with Microsoft Teams. 
+Some example Automation scripts were created and explained on [GitHub](https://github.com/SkylineCommunications/ChatOps-Extensions/blob/main/ChatIntegrationExamples) to get you up to speed. 
```

Comment:
Please place an empty line in front of this paragraph, so it doesn't get combined with the previous one.

```suggestion
To get started, you can find several example Automation scripts with more information on [GitHub](https://github.com/SkylineCommunications/ChatOps-Extensions/blob/main/ChatIntegrationExamples).
```